Homework 17

TetianaZaikina opened the pull request:
Dear teacher!
Thank you very much for checking and helping me. I have rewatched 4 times the lesson , applied the corrections but still got 2 issues (homework17 name and email info). I tried to rename it ( did not help) and debugged ( test stopped at email info, though it is correct). What is my mistake?
Thank you in advance.
Tatiana

TetianaZaikina committed:
Update LoginTests.java

TetianaZaikina committed:
Merge branch 'main' of https://github.com/TetianaZaikina/A49-QA-Automation-Framework

TetianaZaikina committed:
Homework_17

TetianaZaikina committed:
homework_17

-renamed as it throws back an error (did not help);
-applied new changes  but still have issues with email
-debugged and it stopped on email field

Berichicko commented on src/test/java/homework17.java:
```diff
@@ -0,0 +1,66 @@
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+public class homework17 extends BaseTest {
```
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example public class **HomeWork17**

Berichicko commented:
> Dear teacher! Thank you very much for checking and helping me. I have rewatched 4 times the lesson , applied the corrections but still got 2 issues (homework17 name and email info). I tried to rename it ( did not help) and debugged ( test stopped at email info, though it is correct). What is my mistake? Thank you in advance. Tatiana

-

Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/c4d089bb-a37f-4144-a70e-ebcfc55fb067)

**HINT**

**Step 1**

Add `driver.get(url);`

![image](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/e47b61bb-1794-484a-857d-17eb0d195c6e)

**Step 2**

![image](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/491b7c26-1954-47f2-919b-9f743357db83)

Solution:

![Test status_ru](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/9684d160-9b98-476b-a2f1-e1ee695843b5)

For example

![Test failed_ru](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/1dc10e97-d538-4657-a9f7-11eebc89959c)

![image](https://github.com/TetianaZaikina/A49-QA-Automation-Framework/assets/104761482/83a3df8c-8690-4886-be7e-b75d82af5cdb)

TetianaZaikina committed:
HomeWork17

-updated name
-added driver.get (url)

The song is added to playlist, but test fails. I see the difference i assertion, but cannot understand how to correct.
